x-pack/plugins/aiops/server/routes/log_categorization.ts

```diff
@@ -32,6 +36,13 @@ export const defineLogCategorizationRoutes = (
         },
       },
       async (context, request, response) => {
+        const { headers } = request;
+        trackAIOpsRouteUsage(
```

Comment:
@jgowdyelastic is this in the right place? We want to track the number of times the analysis is run from the ML AIOps Labs Log Pattern Analysis page, or from within Discover.

Reply:
This endpoint will be run whenever the analysis is run, but it also has the risk that it could be called without the categorization analysis being run.

Debugging this, it appears there is a bug. The `AIOPS_TELEMETRY_ID.AIOPS_ANALYSIS_RUN_ORIGIN` header is not being added to the request and so the telemetry is not being counted.

Reply:
One alternative would be to also implement client side telemetry to count this on click of the analyse button or in the case of the Discover embedded version, on render of the component.

Reply:
Fixed categorization tracking in c973e760a6ad0fc945cbe11b91ef798903aca1b5